app/pandas.ipynb

```diff
@@ -208,18 +350,92 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "outputs": [],
    "source": [
-    "# Write your code here"
+    "df[\"Average age\"] = df[\"Age\"].mean()\n",
```

Comment:
The 'Average age' column is being added as a constant value for each row, which might not be the intended behavior. If you want to calculate the average age for the entire DataFrame, consider storing it as a separate variable or displaying it separately instead of adding it as a column.